Solar.xcodeproj/project.pbxproj

```diff
@@ -431,7 +432,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
I realize Xcode 12 is recommending iOS 12 as a minimum deployment target, but that’s pretty aggressive considering that quite a few applications are still on iOS 10 or 11.

This Xcode project only affects Carthage. SPM and CocoaPods declare minimum deployment targets in Package.swift and Solar.podspec, respectively, so the minimum deployment target is inconsistent across package managers.

(Full disclosure: I work on the Mapbox Navigation SDK, which still supports iOS 10 and above. We know usage of iOS 10 and 11 is minimal, but as long as applications are declaring support for those versions, SDKs are inclined to support them.)